### ISSUE TITLE
how ta activate shuffle for DataLoader?

### ISSUE THREAD

**Issue by TheZaind** [lucidrains/imagen-pytorch#361]
Yeah, the question is in the title. How can I shuffle my Trainingdata when I am training the model?
Normal, It's something like :
"'shuffle': True"
or
"data_loader = DataLoader(dataset, batch_size=batch_size, shuffle=True)" but i dnt got a DataLoader in this way.
But I don't know where and how to add this.
Anyone an idea?
Thanks 👍
